src/main/peripherals/Lock.java

```diff
@@ -0,0 +1,46 @@
+package peripherals;
+import java.io.IOException;
+import java.net.InetAddress;
+
+
+
+public class Lock implements IPeripheral{	
```

Comment:
I am confused, users will have to change your code in order to add new peripherals?